doc/Makefile.am

```diff
@@ -52,38 +64,103 @@ man-page-build: upg-page-build man-page-start
 		echo "      <tab type=\"user\" visible=\"yes\" url=\"@ref man_$${BASE}\" title=\"$${MANUAL} - $${SUMMARY}\" intro=\"\"/>" >> $(top_builddir)/doc/insert_file ;\
 		## Update the summary man page
-		echo "   <tr id=\"row_$${ID}_\"$${ROW_EVEN}>" >> $(top_builddir)/doc/man_tmp/manpage.dox ;\
-		echo "   <td class=\"entry\" align=\"left\"> @ref man_$${BASE} </td><td class=\"desc\" align=\"left\">$${MANUAL} - $${SUMMARY}</td>" >> $(top_builddir)/doc/man_tmp/manpage.dox ;\
-		echo "   </tr>" >> $(top_builddir)/doc/man_tmp/manpage.dox ;\
+		echo "   <tr id=\"row_$${ID}_\"$${ROW_EVEN}>" >> $(top_builddir)/doc/man_tmp/man_summary.dox ;\
+		echo "   <td class=\"entry\" align=\"left\"> @ref man_$${BASE} </td><td class=\"desc\" align=\"left\">$${SUMMARY}</td>" >> $(top_builddir)/doc/man_tmp/man_summary.dox ;\
+		echo "   </tr>" >> $(top_builddir)/doc/man_tmp/man_summary.dox ;\
 		if [ -z $${ROW_EVEN} ] ; then \
 			ROW_EVEN=" class=\"even\"" ;\
 		else \
 			ROW_EVEN= ;\
 		fi \
 	done ;\
-	## Close off the man page file
-	echo '   </table>' >> $(top_builddir)/doc/man_tmp/manpage.dox ;\
-	echo ' */' >> $(top_builddir)/doc/man_tmp/manpage.dox ;\
+	##
+	## Close off the man page summary file
+	##
+	echo '   </table>' >> $(top_builddir)/doc/man_tmp/man_summary.dox ;\
+	echo ' */' >> $(top_builddir)/doc/man_tmp/man_summary.dox ;\
+	echo '     </tab>' >> $(top_builddir)/doc/insert_file ;\
+	##
+	## Build the individual man pages
+	##
+	echo '     <tab  type="usergroup" visible="yes" url="@ref man_individual" title="Individual Man Pages">' >> $(top_builddir)/doc/insert_file ;\
+	for FILE in $${MAN_FILES} ; do \
+		BASE=`basename $${FILE} | cut -d. -f 1` ;\
+		LIST=`${SED} -ne '/^NAME/,/^SYNOPSIS/p;/^SYNOPSIS/q' $${FILE} | ${SED} -ne '/coap_/{ s/ *, *//g ; p }' | egrep -v "^$${BASE}$$"` ;\
+		for ENTRY in $${LIST} ; do \
+			MANUAL="$${ENTRY}(3)" ;\
+			## Build the manual insert page
+			echo "/// @page man_$${ENTRY} $${MANUAL}" > $(top_builddir)/doc/man_tmp/$${MANUAL}.dox ;\
+			echo "/// @htmlinclude $${ENTRY}.html $${MANUAL}" >> $(top_builddir)/doc/man_tmp/$${MANUAL}.dox ;\
+			## Create html file
+			cat $(top_builddir)/man/$${BASE}.html | ${SED} "s/Function: $${ENTRY}(/<a class=\"anchor\" id=\"$${ENTRY}\"><\/a>\0/" | ${SED} "s/\($${ENTRY}\)\([<(\*<]\)/<span class=\"man-highlight\">\1<\/span>\2/g" > $(top_builddir)/doc/man_html/$${ENTRY}.html ;\
+			## Update scratch_insert_file for sorting later
+			echo "$${ENTRY}" >> $(top_builddir)/doc/scratch_insert_file ;\
+		done ;\
+	done ;\
+	##
+	## Process the (sorted) list of individual man pages
+	##
+	for ENTRY in `cat $(top_builddir)/doc/scratch_insert_file | ALL=C sort -u` ; do \
+		## Update the individual man page
+		MANUAL="$${ENTRY}(3)" ;\
+		echo "      <tab type=\"user\" visible=\"yes\" url=\"@ref man_$${ENTRY}\" title=\"$${MANUAL}\" intro=\"\"/>" >> $(top_builddir)/doc/insert_file ;\
+		echo "   <tr id=\"row_$${ID}_\"$${ROW_EVEN}>" >> $(top_builddir)/doc/man_tmp/man_individual.dox ;\
```

Comment:
The id attributes for the tr element generated for the .dox files (and consecutively for the generated .html) seem to have empty `ID` and `ROW_EVEN` values, resulting in every id being `row__`.

This seems unrelated to the changes in this PR, however.

Reply:
I'm not entirely sure why these id= are needed at all - they are there in the other overview files, but a quick check does not find any usage of them - to get to the particular row in question.  I will remove them.